projects/disagreement/run.py

```diff
@@ -158,14 +157,15 @@ def make_env_all_params(rank, args):
     device = torch.device(dev_name)
     print("device: " + str(device))
 
-    if not run_args.debugging:
```

Comment:
Do we want to save models in all the debugging runs? Maybe we should still add a flag to not save models because that could litter the results folder pretty quickly.